packages/trader/src/Modules/Account/Sections/Profile/FinancialAssessment/financial-assessment.jsx

```diff
@@ -177,6 +177,8 @@ class FinancialAssessment extends React.Component {
                 setStatus({ msg: data.error.message });
             } else {
                 this.setState({ is_submit_success: true });
+                setTimeout(() => this.setState({ is_submit_success: false }), 3000);
```

Comment:
What's the reason behind adding the timeout and updating the state after?

Reply:
This is to change submit button from success state to normal state 3 seconds after submission. So that, after submission it doesn't stay in success state forever